frontend/src/pages/EmailConfirmPage.js

```diff
@@ -19,9 +19,7 @@ export default function EmailConfirmPage() {
 			if (res){
 				setWasConfirmed(true);
 			}
-		}).catch(r =>
-			console.log(r)
-		);
+		}).catch(() => {});
```

Comment:
Why was this removed? Just ran into an error regarding the mail verification and had no information on that at all...

Reply:
I added it back for now. I removed it as I thought we do not want to have anything written to the console when we deploy the frontend somewhere, but for now I think it makes sense to keep it in.